application/src/main/java/com/softserve/lv460/application/security/dto/JWTUserRequest.java

```diff
@@ -13,6 +13,6 @@
 public class JWTUserRequest {
   @Email
   private String email;
-  @Size(min = 8,max =30)
+  @Size(min = 8)
```

Comment:
Are you sure you don't need 'max'?

Reply:
yes, because IDK the size of encrypted password